docs/pbc/all/merchant-management/202407.0/marketplace/import-and-export-data/import-file-details-merchant-comission.csv.md

```diff
@@ -0,0 +1,54 @@
+
+## Import file dependencies
+
+
+
+## Import file parameters
+
+| COLUMN                        | REQUIRED | DATA TYPE | DATA EXAMPLE                                      | DATA EXPLANATION                                |
+|-------------------------------|----------|-----------|---------------------------------------------------|-------------------------------------------------|
+| key                           | ✓        | String    | mc1                                               | Unique identifier of the merchant commission.          |
+| name                          | ✓        | String    | Merchant Commission 1                             | Name of the merchant commission.                |
+| description                   |          | String    |                                                   | Description of the merchant commission.         |
+| valid_from                    |          | Date      | 2024-01-01                                        | Starting from this date the commission is applied. |
+| valid_to                      |          | Date      | 2029-06-01                                        | The last day of when the commission is applied.   |
+| is_active                     | ✓        | Boolean      | 1                                                 | Defines if the merchant commission is active.   |
+| amount                        |          | Integer       | 5                                                 | Amount of the merchant commission.              |
+| calculator_type_plugin        | ✓        | String    | percentage                                        | Type of the calculator plugin used to calculate the commission.             |
+| merchant_commission_group_key | ✓        | String    | primary                                           | Defines the merchant commission group. Can be primary or secondary.           |
+| priority                      | ✓        | Integer       | 1                                                 | Priority of the merchant commission.            |
+| item_condition                |          | String    | item-price >= '500' AND category IS IN 'computer' | Item conditions that must be fulfilled to apply the commission.                         |
+| order_condition               |          | String    | "price-mode = ""GROSS_MODE"""                     | Condition for the order.                        |
+| merchants_allow_list |       |  It contains a list of merchant references separated by commas, such as “MER000002,MER000006”
+| fixed_amount_configuration |    |     |    EUR|0.5|0.5,CHF|0.5|0.5         |     Defines fixed amount commission configuration in case a fixed commission needs to be applied to each item in the order. Format: `CURRENCY|GROSS AMOUNT|NET AMOUNT` |
```

Comment:
Do you know why these two last fields are not in the installation guide? See https://github.com/spryker/spryker-docs/blob/master/_includes/pbc/all/install-features/202407.0/marketplace/install-the-marketplace-merchant-commission.md#5-import-data

Reply:
Because import data from BO and from import folder are different formats. Basically in BO its a single file while in import folder its several files.

Reply:
Those files have different requirements for data, e.g. amounts imported via import folder are multiplied by 100 while amounts imported from BO are not.

100.99$ in BO is 100.99
100.99$ in Import is 10099

Reply:
We need to have two separate descriptions for data import here.